package.json

```diff
@@ -185,7 +180,7 @@
     "rimraf": "2.6.2",
     "run-sequence": "2.2.0",
     "script-ext-html-webpack-plugin": "1.8.7",
-    "semantic-release": "8.2.0",
+    "semantic-release": "12.2.2",
```

Comment:
I think I had a issue with the newer version of semantic-release. But that was a while ago, don't remember what was the cause.

Reply:
Must have been the pre/post steps, I'd guess, which aren't required anymore.
I had to refactor the release step a little bit to make it work with our `build > release` routines.